src/main/java/com/example/WorldOfAnimals/dto/AnimalDTO.java

```diff
@@ -0,0 +1,33 @@
+package com.example.WorldOfAnimals.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.sql.Timestamp;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class AnimalDTO {
+
+    private Long id;
+
+    private String name;
+
+    private String description;
+
+    @JsonProperty(value = "animalType")
+    private AnimalTypeDTO animalTypeDTO;
```

Comment:
Maybe rename to animalType ? and delete @JsonProperty(value = "animalType")

Reply:
done